ntc_templates/templates/cisco_xr_show_interfaces_description.textfsm

```diff
@@ -0,0 +1,7 @@
+Value INTERFACE (\S+)
+Value STATUS (up|down|deleted|admin\-down|reset)
+Value PROTOCOL (up|down|deleted|admin\-down)
+Value DESCRIPTION (\S.*\s*\S*)
+
+Start
+  ^${INTERFACE}\s+${STATUS}\s+${PROTOCOL}(?:\s+)?(?:${DESCRIPTION})? -> Record
```

Comment:
```suggestion
  ^${INTERFACE}\s+${STATUS}\s+${PROTOCOL}\s*(?:${DESCRIPTION})?\s*$$ -> Record
  ^\S+\s+\S+\s+\d+
  ^Interface\s+Status\s+Protocol\s+Description\s*$$
  ^-+\s*$$
  ^\s*$$
  ^. -> Error
```

Reply:
I think we can simplify catching description by matching end of line.

We also require that all new templates Error out when lines are unmatched.